compile.sh

```diff
@@ -212,13 +212,14 @@ do
         echo "Invalid number of cores was specified: $2"
         exit 1
       fi
-      if [ "$MAXCORES" -ge "$2" -o "$MAXCORES" -eq 0 ]; then
+      if [ "$MAXCORES" -ge "$2" -o "$MAXCORES" -eq 0 ] || [ -n "$DISTCC_POTENTIAL_HOSTS" ]; then
```

Comment:
```suggestion
      if [ "$MAXCORES" -ge "$2" -o "$MAXCORES" -eq 0 -o -n "$DISTCC_POTENTIAL_HOSTS" ]; then
```
I think…

Yes, this is much better.